src/WebJobs.Extensions/Extensions/Timers/Scheduling/ScheduleMonitor.cs

```diff
@@ -14,7 +14,12 @@ namespace Microsoft.Azure.WebJobs.Extensions.Timers
     /// </summary>
     public abstract class ScheduleMonitor
     {
-        private static DateTime defaultDateTime = new DateTime(0, DateTimeKind.Local);
```

Comment:
This was ultimately the bug

Reply:
![argh](https://media3.giphy.com/media/v1.Y2lkPTc5MGI3NjExNm9na2VlcmM4N2V3cWRoOG9jaXYxMnNsdmJ6cXkyY2E1aGV6bzZmciZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/d3EdJTouFN890fMnGP/giphy.gif)